### ISSUE TITLE
App crash when calling signIn 

### ISSUE THREAD

**Issue by GyuriMajercsik** [Abedalkareem/games_services#15]
When integrating this plugin the application crashes with below error when calling:
`GamesServices.signIn()`

```
com.google.android.gms.common.api.ApiException: 4: 4: 
        at com.google.android.gms.common.internal.ApiExceptionUtil.fromStatus(com.google.android.gms:play-services-base@@17.1.0:4)
        at com.google.android.gms.common.internal.zai.zaf(com.google.android.gms:play-services-base@@17.1.0:2)
        at com.google.android.gms.common.internal.zak.onComplete(com.google.android.gms:play-services-base@@17.1.0:6)
        at com.google.android.gms.common.api.internal.BasePendingResult.zaa(com.google.android.gms:play-services-base@@17.1.0:176)
        at com.google.android.gms.common.api.internal.BasePendingResult.setResult(com.google.android.gms:play-services-base@@17.1.0:135)
        at com.google.android.gms.auth.api.signin.internal.zzi.zzc(com.google.android.gms:play-services-auth@@18.0.0:5)
        at com.google.android.gms.auth.api.signin.internal.zzs.dispatchTransaction(com.google.android.gms:play-services-auth@@18.0.0:6)
        at com.google.android.gms.internal.auth-api.zzc.onTransact(com.google.android.gms:play-services-auth@@18.0.0:13)
        at android.os.Binder.execTransactInternal(Binder.java:1032)
        at android.os.Binder.execTransact(Binder.java:1005)
```

I was struggling to fix this issue, considering that the problem is in my app ( SHA-1 missing or something else ) but I didn't find anything for two days. In my desperation I tried the other plugin [play_games](https://pub.dev/packages/play_games) and it worked. Although the logs shows the same error it says that: "Failed to silent signin, trying explicit signin". 

Basically, this error is handled in the other plugin or maybe this shouldn't appear from beginning and I'm doing something wrong ? 

p.s.: thanks for the plugin

**Comment by Abedalkareem** [comment]
Hi,
Fixed. Thank you for reporting.
